cylc/flow/cfgspec/suite.py

```diff
@@ -534,12 +534,43 @@
                 Example Recurrences:
 
                 date-time cycling:
-                   ``T00,T06,T12,T18`` or ``PT6H`` - *every six hours*
+                   * ``R1`` - once at the intial cycle point
+                   * ``T00,T06,T12,T18`` - daily at 00:00, 06:00, 12:00
+                     & 18:00
+                   * ``PT6H`` - every six hours starting at the initial
+                     cycle point
                 integer cycling:
-                   ``P2`` - *every other cycle*
+                   * ``R1`` - once at the intial cycle point
+                   * ``P2`` - every other cycle
+                   * ``P3,P5`` - every third or fifth cycle
```

Comment:
If you build this as is the formatting of `integer cycling` and `date-time cycling` is the same as that for config item.
```suggestion
                date-time cycling:

                   * ``R1`` - once at the intial cycle point
                   * ``T00,T06,T12,T18`` - daily at 00:00, 06:00, 12:00
                     & 18:00
                   * ``PT6H`` - every six hours starting at the initial
                     cycle point

                integer cycling:

                   * ``R1`` - once at the intial cycle point
                   * ``P2`` - every other cycle
                   * ``P3,P5`` - every third or fifth cycle
```

Reply:
Yeah, thats a regression bug in the rtd theme I need to poke them about at some point (it effects other things, mainly the API docs).

Your diff above converts the bullet point lists into block quotes. Would rather keep the RST correct and fix the particular theme with the issue:

```rst
definition list item:
    definition list description

plain text:

   blockquote
```

Reply:
Here's the bug report - https://github.com/readthedocs/sphinx_rtd_theme/issues/800, https://github.com/readthedocs/sphinx_rtd_theme/issues/979

It was fixed a while back but seems to be back.

Reply:
Digging a little deeper I think it is because the definition list is being documented within a domain.

Take this example:

```rst
Foo
   foo
Bar
   bar
Baz
   baz


.. py:class:: pub

   Arguments:
      foo: abc
      bar: bcd
      baz: def

   Foo
      foo
   Bar
      bar
   Baz
      baz
```

Which renders like this:

<img width="267" alt="Screenshot 2021-02-12 at 12 37 55" src="https://user-images.githubusercontent.com/16705946/107769011-35e4a700-6d2f-11eb-9f24-45c5c6f587cc.png">

Reply:
Yep that's it - https://sphinx-rtd-theme--981.org.readthedocs.build/en/981/demo/api.html#test_py_module.test.Foo

Reply:
Raised an issue - https://github.com/readthedocs/sphinx_rtd_theme/issues/1052